README.md

```diff
@@ -1 +1,48 @@
-# bookstore
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Bookstore
+
+This website will display a list of books, with their respective detail information such as autor, title, gender, chapitres. You can add and erase elements.
+
+## Live Version
+Under construction
+
+## Built With
+
+- HTML&CSS
+- Javascript
+- React
+
+## Getting Started
+To get a local copy up and running follow these simple steps.
+
+### Setup
+ 
+
+### Install
+- Open your console and change the current working directory into the destination for this project.
+- Type `git clone git@github.com:JasemDuncan/bookstore.git`
+- Type Enter to finish the installation process.
+
```

Comment:
- [ ] Technically, users cannot run your app locally without running `npm i` and `npm start`. Kindly add these commands as necessary for setting up and running the application.